graylog2-web-interface/src/components/alertconditions/AlertConditionForm.jsx

```diff
@@ -8,26 +8,44 @@ const AlertConditionForm = React.createClass({
     type: React.PropTypes.string.isRequired,
     alertCondition: React.PropTypes.object,
   },
-  alertConditionsFactory: new AlertConditionsFactory(),
+  getDefaultProps() {
+    return {
+      alertCondition: {
+        parameters: {},
+      },
+    };
+  },
   getValue() {
-    return this.refs.conditionForm.getValue();
+    return {
+      title: this.refs.title.value,
+      parameters: this.refs.conditionForm.getValue(),
+    };
   },
+  alertConditionsFactory: new AlertConditionsFactory(),
+
   _formatConditionFormFields(type) {
     const typeDefinition = this.alertConditionsFactory.get(type);
 
     if (typeDefinition !== undefined) {
-      return <typeDefinition.configuration_form ref="conditionForm" alertCondition={this.props.alertCondition}/>;
+      return <typeDefinition.configuration_form ref="conditionForm" alertCondition={this.props.alertCondition.parameters}/>;
     }
 
     return undefined;
   },
   render() {
+    const alertCondition = this.props.alertCondition || { parameters: {} };
```

Comment:
At the moment this default alert condition will never be used, as the `getDefaultProps` method will already give us a default alert condition.